index.js

```diff
@@ -0,0 +1,214 @@
+import * as Sentry from '@sentry/node'
+import getRawBody from 'raw-body'
+import httpAssert from 'http-assert'
+import * as ethers from 'ethers'
+import { json, status } from 'http-responders'
+import assert from 'node:assert'
+
+const maxScore = BigInt(1e15)
+// https://github.com/filecoin-station/spark-impact-evaluator/blob/fd64313a96957fcb3d5fda0d334245601676bb73/test/Spark.t.sol#L11C39-L11C65
+const roundReward = 456621004566210048n
```

Comment:
Should we document what is the unit of the round reward values?

Reply:
Any crypto value is typically stored in its lowest unit (attoFIL, wei), as these are integers, without decimal places. I would say this isn't specific to this service and should be done everywhere. I don't mind adding a unit if you prefer, eg `roundRewardAttoFIL`?